lib/options.go

```diff
@@ -44,6 +45,86 @@ const DefaultScenarioName = "default"
 // nolint: gochecknoglobals
 var DefaultSummaryTrendStats = []string{"avg", "min", "med", "max", "p(90)", "p(95)"}
 
+type DNSConfig struct {
+	TTL      null.String     `json:"ttl"`
+	Strategy NullDNSStrategy `json:"strategy"`
+	// FIXME: Valid is unused and is only added to satisfy some logic in lib.Options.ForEachSpecified(),
+	// otherwise it would panic with `reflect: call of reflect.Value.Bool on zero Value`.
+	Valid bool
+}
+
+//go:generate enumer -type=DNSStrategy -transform=kebab -trimprefix DNS -output dns_strategy_gen.go
+type DNSStrategy uint8
+
+type NullDNSStrategy struct {
+	DNSStrategy
+	Valid bool
+}
+
+const (
+	DNSFirst DNSStrategy = iota + 1
+	DNSRoundRobin
+	DNSRandom
+)
+
+const DefaultDNSConfigText = "ttl=inf,strategy=first"
```

Comment:
I was lazy about it, but this should probably be returned by `DNSConfig.MarshalText()`, right?

Reply:
Sure, implementing `String()` sounds better, thanks.

Reply:
Resolved by 53c94b92.